docs/index.rst

```diff
@@ -15,9 +15,9 @@ Specutils Documentation
 .. image:: img/logo.png
 
 
-The ``specutils`` package provides a basic interface for the loading,
-manipulation, and common forms of analysis of spectroscopic data. These
-generic data containers and accompanying modules will provide a toolbox that the
+``specutils`` is a Python package for the representation of, loading,
```

Comment:
What you have is grammatically correct, but the mixing of nouns (representation) and gerunds (loading) just always seemed a bit weird to me. Ignore if you feel otherwise!